### ISSUE TITLE
sensitivity

### ISSUE THREAD

**Issue by zakhaj** [imbushuo/mac-precision-touchpad#48]
even on the highest sensitivity under touchpad settings... on a MBPr retina late 2012....
doesn't seem to always register when touched lightly.
for example, tips of fingers isn't as responsive.  works well with the 'pads' of finger.

otherwise a fantastic piece of work and huge credit to those responsible. its great, I love it.

**Comment by imbushuo** [comment]
I am thinking about exposing an interface for sensitivity control, but it is under consideration yet.

**Comment by icstyle** [comment]
@imbushuo Yes, please!! Would be so great!

**Comment by TGSAN** [comment]
@imbushuo Yes, using my fingertips cannot control the touchpad in MacBook Pro 2014 Mid. But BootCamp driver have no problem. ( Google Translate )

**Comment by battaglia01** [comment]
Is there any way to change this?
